src/blog/2023/011-form-js-1.6.md

```diff
@@ -0,0 +1,101 @@
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-dynamic-list-simple-scope.gif">
+  <p class="caption">
+    Conditionally hide or show on a row by row basis within a dynamic list. 
+  </p>
+</div>
+
+So, while simple on the surface, the dynamic list really was the final piece of the puzzle for a great deal of applications. The ability to flexibly handle data arrays combined with binding paths and keys leads to make quick configuration and clean data outputs the default experience in form-js.
+
+Now, what if you're dealing with much larger data sets that you want to render in the application, browse through, maybe use as a reference within your form?
+
+
+## The first iteration of Tables
+
+If you need to display large datasets, for reference purpose for example, you may use the new table component.   
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-tables.gif">
+  <p class="caption">
+    Efficiently manage tabular data with the new Table Component.
+  </p>
+</div>
+
+Our first iteration already brings in plenty of functionality, namely row sorting, pagination with control over each page's height, and both static and dynamic definition of headers which means you have the ability to render your tables entirely from input data definitions, giving you a great deal of programatic control over the final appearance.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-tables-more.png">
+  <p class="caption">
+    We're not done yet with this component. We will iterate upon tables based on feedback, and see where we can take it.
+  </p>
+</div>
+
+
+## Our focus for 1.7
+
+As you've probably noticed, form-js has grown a lot in the last year or two. We've made a lot of experimental calls, and built a lot of new things from scratch. But we haven't had dedicated time to step off the gas yet, which has caused some performance slowdowns and some of the usual problems in growing projects. 
+
+So, we've decided not to focus on any major features for 1.7, but instead to polish, get rid of architectural pains in the code base and improve the performance of form-js rendering.
+
+So if you're one of those folks rendering forms with 1000+ elements and struggling, be on the lookout for this next release, this one's for you.
+
+Happy holiday season everyone!
+
+
+# Wrapping Up
+
+To discover all the enhancements in form-js 1.6 and recent versions, check out our [full changelog](https://github.com/bpmn-io/form-js/blob/master/packages/form-js/CHANGELOG.md#140).
+
+We are always looking for feedback, so if you've spotted a bug, have an improvement to suggest, or just a related thought really, then don't be shy and reach out to us via [our forums](https://forum.bpmn.io/), tweet us [@bpmn_io](https://twitter.com/bpmn_io), or file any issue you found in [the form-js issue tracker](https://github.com/bpmn-io/form-js/issues).
```

Comment:
We switched from Twitter to Mastodon, cf. other blogs, and need to adjust accordingly.